velox/exec/Window.cpp

```diff
@@ -413,12 +420,14 @@ void fixOutOfBoundFrames(
     const vector_size_t& numRows,
     vector_size_t* frameStart,
     vector_size_t* frameEnd) {
-  for (auto i = frameStart, j = frameEnd; i < frameStart + numRows; i++, j++) {
+  for (auto i = 0; i < numRows; i++) {
+    auto frameStartIndex = frameStart + i;
```

Comment:
Please move the variable definition outside the loop so that its not constructed each time.

Reply:
Updated, thanks!